.github/workflows/e2e.yml

```diff
@@ -11,9 +11,9 @@ on:
 env:
   K8S_VERSION: "v1.27.3"
   KIND_VERSION: "v0.20.0"
-  KIND_CLUSTER_NAME: "clustercidr"
+  KIND_CLUSTER_NAME: "nodeipam"
   REGISTRY: ghcr.io
-  IMAGE_NAME: mneverov/cluster-cidr-controller
+  IMAGE_NAME: sigs.k8s.io/node-ipam-controller
```

Comment:
This is invalid. there is no registry `sigs.k8s.io` unless you want to do a local build ?